sway-lib-std/src/alias.sw

```diff
@@ -1,6 +1,4 @@
 library;
 
-/// The `AssetId` type is simply an alias for `b256` that represents the ID of a native asset.
-pub type AssetId = b256;
 /// The `SubId` type is simply an alias for `b256` that represents the sub identifier of a native asset.
 pub type SubId = b256;
```

Comment:
Should we do the same thing for `SubId` - wrapping it in a struct?

Reply:
No, not necessary, theres no methods that are required on it, so the ease of use of it being a b256 alias is preferred

Reply:
I've had various thoughts about this before.

I think that we ought to look for ways to provide meaning to turn the alias into a formalized structure.
I think it's better if we start to lead people away from using the `b256` directly and use wrapper methods... but we need to have meaningful methods for them to use.

Having an `alias` file is debatable. We may have it akin to the `constants` file but at the same time having an alias defined in the file where the aliased object exists also makes sense. Without conditional compilation it's probably preferable to have them defined in different files, with conditional compilation having a separate file isn't an issue.